conda_smithy/configure_feedstock.py

```diff
@@ -1198,6 +1199,16 @@ def render_drone(jinja_env, forge_config, forge_dir, return_metadata=False):
     )
 
 
+def render_actions(jinja_env, forge_config, forge_dir, render_info=None):
+    # this file was copied over w/ all of the feedstock content
+    # we have to remove it if the actions are not on
+    if not forge_config.get("bot", {}).get("automerge", False):
+        try:
+            remove_file(f"{forge_dir}/.github/workflows/main.yml")
+        except Exceptions:
```

Comment:
Don't use generic Exception. You need to be specific.

Reply:
Done!